README.md

```diff
@@ -166,11 +166,16 @@ For nvim-cmp, it is also possible to follow the [example recommendation](https:/
 
 ## Add Snippets
 
-- **Vscode-like**: For using snippets from a plugin (eg. [rafamadriz/friendly-snippets](https://github.com/rafamadriz/friendly-snippets)) install it and add
+- **Vscode-like**: To use existing vs-code style snippets from a plugin (eg. [rafamadriz/friendly-snippets](https://github.com/rafamadriz/friendly-snippets)) simply install the friendly-snippets plugin and then add
```

Comment:
"the friendly-snippets plugin" -> "the plugin"

Just to keep it general, I could see that as a source of confusion if someone uses a different plugin, maybe they'd assume this only works for `friendly-snippets`

Reply:
yep you're right. Maybe I'm just in love friendly-snippets : )